deliver/README.md

```diff
@@ -36,7 +36,7 @@ deliver
 
 ###### Upload screenshots, metadata and your app to the App Store using a single command
 
-`deliver` can upload ipa or pkg files, app screenshots and more to iTunes Connect from the command line.
+`deliver` will upload screenshots, metadata and binaries to iTunes Connect. It’s what you use to submit your app for review.
```

Comment:
"`deliver` uploads" to match the style of the other README's. Then maybe "Use deliver to submit your app for review."?